sql/snowflake/SnowflakeParser.g4

```diff
@@ -2657,6 +2658,7 @@ wh_extra_size
 
 wh_properties
     : WAREHOUSE_SIZE EQ ( wh_common_size | wh_extra_size | ID2)
+    | WAREHOUSE_TYPE EQ (STANDARD | SNOWPARK_OPTIMIZED)
```

Comment:
It can be merged with the previous alternative:

```suggestion
    : WAREHOUSE_SIZE EQ (wh_common_size | wh_extra_size | ID2 | STANDARD | SNOWPARK_OPTIMIZED)
```

Reply:
how can this be merged, these are 2 different token (SIZE and TYPE)

Reply:
Sorry, I missed that.